modules/installation-azure-regions.adoc

```diff
@@ -34,6 +34,7 @@ The installation program dynamically generates the list of available Microsoft A
 * `japanwest` (Japan West)
 * `koreacentral` (Korea Central)
 * `koreasouth` (Korea South)
+* `mexicocentral`  (Mexico Central)
```

Comment:
```suggestion
* `mexicocentral` (Mexico Central)
```
Looks like there is an extra space here :)

Reply:
I removed the extra space - thanks.